### DISCUSSION TITLE
[잡담] 내일까지 과제 제출인데....

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#105] [잡담]
과제 어렵나유?

후기 궁금...ㅎㅎ

물론 쉽다고 말하면 다음 과제는 어렵게 가고
어렵다고 말하면 다음 과제는 어렵게 갈거니까 걱정마요 ㅎㅎ

**Comment by lsj0822** [comment]
방금끝냈는데, 지난주보다 어려웠어요... 보기엔 더 쉬워보였는데
